src/components/CardList.js

```diff
@@ -0,0 +1,51 @@
+import "./CardList.css";
+
+export default function CardList({ url, createdAt, desc, imgUrl }) {
+  const linkUrl = () => {
+    window.open(url);
+  };
+
+  const formattedDate = new Date(createdAt)
+    .toLocaleString("ko-KR", {
+      year: "numeric",
+      month: "2-digit",
+      day: "2-digit",
+    })
+    .replace(/\./g, ". ");
+
+  const timeAgo = (createdAt) => {
+    const currentTime = new Date();
+    const createdTime = new Date(createdAt);
+    const timeDiff = Math.abs(currentTime - createdTime);
```

Comment:
엄밀히따지면 timeDiff가 -값이 나온다면(매개변수로 받은 시간이 현재 시간 이후면) 에러를 발생시켜줘야 할 것 같아요~
timeAgo라는 함수는 매개변수로 넘어오는 날짜가 현재 시간보다 이전이라는 걸 가정하고 있기 때문이에요!

Reply:
따라서 abs와 같이 절대값을 리턴해주는 메소드가 아닌 소수점을 제거해주는 메소드를 사용해주는게 맞을 것 같아요!
에러처리를 해준다면 이렇게 할 수 있구요~
```suggestion
    const timeDiff = Math.trunc(currentTime - createdTime);
    if(timeDiff < 0) {
      throw Error('현재 시간보다 이전 시간만 입력할 수 있습니다');
      // 또는
      console.warn('현재 시간보다 이전 시간만 입력할 수 있습니다');
      return;
    }
```